neural-network/PulsedRADAR/README.md

```diff
@@ -0,0 +1,58 @@
+# Moku Neural Network Example - Pulsed RADAR
+
+The example code in this directory is designed to highlight the following capabilities:
+
+- Application of [autoencoder](https://apis.liquidinstruments.com/mnn/examples/Autoencoder.html) Neural Network Capability
```

Comment:
Maybe link to the autoencoder example here (`../Autoencoder.ipynb`) rather than the html export of the notebook? And below too.

Reply:
Good suggestion.  I've changed the link in both places.